doc/testing.rst

```diff
@@ -0,0 +1,79 @@
+Automating the tests for your App
+=================================
+
+This an introductory guide to writing automated tests for your
+Morepath app.  We assume you've already installed Morepath; if not,
+see the :doc:`installation` section.
+
+In order to carry out the test we'll use WebTest_, which you'll need
```

Comment:
I'd say people would install webtest in their test dependencies of their project, along with pytest. So I think we should make that the primary explanation with a quick & dirty install webtest & pytest alongside.